Gemfile

```diff
@@ -27,7 +27,7 @@ gem "stimulus-rails"
 # Build JSON APIs with ease [https://github.com/rails/jbuilder]
 gem "jbuilder"
 
-gem 'bootstrap', '~> 5.2', '>= 5.2.3'
+gem "bootstrap-sass",  "3.4.1"
```

Comment:
Is there a specific reason you changed the gem to `bootstrap-sass`? That gem is for Bootstrap 3 which is now outdated. The `bootstrap` gem is for Bootstrap 5, and it does work with SASS.